apps/platform/.eslintrc.js

```diff
@@ -1,7 +1,11 @@
 /** @type {import("eslint").Linter.Config} */
 module.exports = {
   root: true,
-  extends: ["@muqa/eslint-config/next.js"],
+  extends: [
+    "@muqa/eslint-config/next.js", 
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended"],
```

Comment:
Move these to `@muqa/eslint-config/next.js` so it's common for other places that extend it.

Reply:
moved, thx